src/main/java/core/basesyntax/converter/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.converter;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try (BufferedWriter bufferedWriter =
+                     new BufferedWriter(new java.io.FileWriter(filePath, false))) {
+            bufferedWriter.write(data);
+            bufferedWriter.newLine();

```

Comment:
why do you need it?

Reply:
I use a BufferedWriter to write the data being transferred